Resources/views/Collector/migrations.html.twig

```diff
@@ -0,0 +1,175 @@
+{% extends app.request.isXmlHttpRequest ? '@WebProfiler/Profiler/ajax_layout.html.twig' : '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% set profiler_markup_version = profiler_markup_version|default(1) %}
+
+    {# this is the content displayed as a panel in the toolbar #}
+    {% set icon %}
+        {% if profiler_markup_version == 1 %}
+            {% set color = collector.executedUnavailableMigrations > 0 ? 'red' : '' %}
```

Comment:
As you put it in red when there are some unavailable migration executed, you should also display this info in the details